public/assets/sass/admin/image-store.scss

```
@@ -0,0 +1,104 @@
+@import "../util/sass-mixins";
+
+#content {
+  max-width: var(--content-width);
+  margin: auto;
+  padding: 0 2em;
```

```suggestion
  padding: 0 2em;
  min-height: 450px;
```
this prevents issues where the footer isn't at the bottom on the screen when the search functionality filters out all results.